tooling/cli/templates/app/src-tauri/capabilities/default.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "./schemas/desktop-schema.json",
+  "identifier": "default-plugins",
+  "description": "enables the default permissions",
+  "windows": ["main"],
+  "permissions": [
+    "path:default",
+    "event:default",
+    "window:default",
+    "webview:default",
+    "app:default",
+    "resources:default",
```

Comment:
do we want to enable this by default? or better can `menu:default` and `tray:default` include this permission? because resources is kinda internal and users should not need to deal with it.

Reply:
right now there's no way for a plugin to enable commands on other plugins.. but we can adapt core to ignore permission checks for the resources plugin worst case

Reply:
is that for a security reason or just not implemented?

Reply:
we can include all core plugin default permissions automatically.. then the user can deny the functions he doesn't want.. what do you think @chippers